pkg/scheduler/framework/plugins/podtopologyspread/scoring_test.go

```diff
@@ -427,8 +427,8 @@ func TestPodTopologySpreadScore(t *testing.T) {
 			},
 			failedNodes: []*v1.Node{},
 			want: []framework.NodeScore{
-				{Name: "node-a", Score: 33}, // +19 compared to maxSkew=1
-				{Name: "node-b", Score: 55}, // +13 compared to maxSkew=1
+				{Name: "node-a", Score: 44}, // +19 compared to maxSkew=1
```

Comment:
Update the code annotation `XX compared to maxSkew=1`  or delete it  ^_^

Reply:
Updated it.